assets/src/blocks/Submenu/BackTop.js

```diff
@@ -0,0 +1,18 @@
+import useScrollPosition from '@react-hook/window-scroll';
+
+export const BackTop = ({}) => {
```

Comment:
No need for the brackets here 🙂 `export const BackTop = () => {` should be fine

Reply:
a yeah that's a leftover :+1:

Reply:
Ended up removing the component in favor of putting it into the base template here: https://github.com/greenpeace/planet4-master-theme/pull/1166/files